### DISCUSSION TITLE
How to define component colors per color scheme in createTheme?

### DISCUSSION THREAD

**Discussion by holm-dk** [mantinedev/mantine#5130] [Q&A] [answered]
I've been struggling trying to find out how to define component colors per color scheme in the function `createTheme`.
I was hoping for something similar to this:

```ts
export const theme = createTheme({
  components: {
    Card: Card.extend({
      styles: (_theme) => ({
        root: {
          backgroundColor: _theme.colorScheme === "light" ? "pink" : "yellow"
        }
      })
    })
  }
});
```

But I can't figure out how to access the current colorScheme - it's not available on the theme object.

Is it possible to achieve this, or is the preferred method to use PostCSS?

**Comment by rtivital** [comment, marked as answer]
If you do not have ssr in your application, you can get color scheme value from html attribute – `document.documentElement.getAttribute('data-mantine-color-scheme')`. If you have ssr, then define styles in CSS modules.

**Reply by holm-dk** [reply]
All right - thank you very much.
I was hoping that it was accessible through function arguments in the styles api, but it'll do :-)
